dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/SmallTestCase/SmallScaleTest.mo

```diff
@@ -0,0 +1,315 @@
+within Dynawo.Examples.Nordic.SmallTestCase;
```

Comment:
I propose to discard the SmallTestCase and not push it into dynawo master. We can keep it in another repo however.

Reply:
Where does it belong ?